src/validation_workflow/validation_args.py

```diff
@@ -109,7 +109,14 @@ def __init__(self) -> None:
             help="Enter type of artifacts that needs to be validated",
             choices=["staging", "production"],
             default="production",
-            dest="artifact_type",
+            dest="artifact_type"
+        )
+        parser.add_argument(
+            "-s",
+            "--allow-without-security",
+            action="store_true",
+            default=False,
+            help="Allow Validation without security"
```

Comment:
This is not a good design here.
-s should be parameter for things like SECURITY_ENABLED (true w/ sec, false w/o sec, default true), instead of `--allow-without-security`, which is very weird and easily get confused.

Reply:
Used -s  as it is related to security, when the above argument is provided by user - we allow validation of artifacts without security plugin, by checking the presence of security plugin in the installed plugins.

Reply:
I mean to say change `--allow-without-security` to something like `--enable-security` or `--security-enabled` instead of the other way around. True should be security enabled and false should be security disabled, instead of true meaning without security.

Reply:
After discussing with @Divyaasm on this topic, it is clear now that what she wants is a parameter such as `--force-https-check`.

If this option is false it will check for security plugin, if present use https, else use http.
If this option is true it will always use https regardless if security plugin present.

It is an option for early release builds that security plugin is not available.

Suggesting changing the name from `--allow-without-security` to `--force-https-check` and from `-s` to `-f`.

Thanks.